### PR TITLE
update/remove docs: READMEs, Usage, and confluence pages

### PR DESCRIPTION
**Pull request by leslie-corbalt**
https://jiraent.cms.gov/browse/CMCSMACD-1874

update confluences pages are [here](https://confluenceent.cms.gov/x/KSanD) (a Guide) and [here](https://confluenceent.cms.gov/x/-Nj_Fw)) (Setting up a self-hosted runner)

### CODE REVIEW

File: github-oidc/README.md [hunk 1]
```diff
@@ -16,15 +22,17 @@ Note: In order to successfully create IAM resources in CMS Cloud, developers mus
...
+An IAM OIDC identity provider is a resource in IAM that describes an external identity provider (IdP) service that supports the [OpenID Connect (OIDC) standard](http://openid.net/connect/). The AWS identity provider resource is configured with the URL (`token.actions.githubusercontent.com`) and server certificate thumbprint of the GitHub OIDC provider. A valid default value for the thumbprint is provided, but thumbprint can also be obtained by following [these steps](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html).
+
+NOTE: AWS secures communication with the GitHub IdP through it's library of trusted root certificate authorities (CAs) instead of using a certificate thumbprint to verify GitHub's IdP server certificate. The thumbprint remains in the AWS identity provider configuration, but is no longer used for validation. source: [AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html)
```
Comment (ben-harvey) [inline github-oidc/README.md:27]: typo: it's -> its

File: README.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
-This repository contains Dockerfiles for self-hosted GitHub Actions runners and an associated Terraform module which can be run in your environment to provision:
+This repository contains Dockerfiles for self-hosted GitHub Actions runners and an associated Terraform module, `github-actions-runner-terraform`, which can be run in your environment to provision:
 
 - ECS Cluster
 - ECS Service
 - ECS Fargate task definition to spin up an instance of this runner _per job_ in your GitHub Actions workflow
 
-This module uses an existing ECR repository in AWS, and so does not provision one.
+This module uses the ECR repository running in the MACBIS Shared DSO Dev account, managed in the `terraform/dev/account` directory, and accessible by the MACBIS organization.
```
Comment (ben-harvey) [inline README.md:11]: nit: ECR repositories don't 'run' per se

File: Usage.md [hunk 3]
```diff
@@ -56,15 +56,15 @@ jobs:
 
   test-self-hosted:
     name: Testing self-hosted tag
-    needs: start-runner
+    needs: start-runners
```
Comment (ben-harvey) [inline Usage.md:59]: Can't comment directly, but might as well bump the version of the `configure-aws-credentials` action to `v4` in the example while whe're here

File: github-oidc/README.md (hunk 1 above)
Comment (leslie-corbalt) [reply github-oidc/README.md:27]: Done.

File: README.md (hunk 2 above)
Comment (leslie-corbalt) [reply README.md:11]: Changed "running" to "create"

File: Usage.md (hunk 3 above)
Comment (leslie-corbalt) [reply Usage.md:59]: Done.

**Review by ben-harvey — APPROVED** [review]
LGTM!

**Comment by leslie-corbalt** [comment]
#195

**Pull request merged**